samples/sensor/wsen_itds/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: WSEN-ITDS Sensor Sample
+tests:
+  sample.sensor.wsen-itds:
+    harness: sensor
+    tags: sensors
+    depends_on: i2c gpio
+    filter: dt_compat_enabled("we,wsen-itds")
```

Comment:
@nashif Will this filter work if the application adds the sensor in a board-specific dts overlay? Or do we need to use a whitelist?

Reply:
@nashif ping